tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php

```diff
@@ -187,7 +191,7 @@ protected function deleteIndexerEntry($userId)
 		{
 			$db->setQuery($query)->execute();
 		}
-		catch (JDatabaseExceptionExecuting $e)
+		catch (DataObjectbaseExceptionExecuting $e)
```

Comment:
DataObjectbaseExceptionExecuting not loaded using `use`?

Reply:
done